mod/metrics/set/type_plus_right/metric/formula/wiki_rating.rb

```diff
@@ -2,7 +2,7 @@
   view :rating_core do |args|
     table_content =
       card.translation_table.map do |metric, weight|
-        [subformat(metric)._render_thumbnail_plain(args), weight + "%"]
+        [subformat(metric)._render_thumbnail_plain(args), "#{weight}%"]
```

Comment:
better use `nest` instead of `subformat`

Reply:
ok, I'll change.  I wasn't really wanting to tinker in this area.  not sure why this broke in tests.